domains.txt

```diff
@@ -705,16 +788,19 @@ tri-executive.com
 triad.co.uk
 triplesevenresources.com
 trustinsoda.com
+tsinternational.co.uk
```

Comment:
TS International Solutions specialises in fulfilling staffing and recruitment needs in Oil & Gas and Engineering disciplines across the world.